src/components/DriverOverview/DriverOverview.tsx

```diff
@@ -0,0 +1,220 @@
+"use client";
+
+import React from "react";
+import { Text, Document, Page, View, StyleSheet, Image } from "@react-pdf/renderer";
+
+export interface DriverOverviewCardProps {
+    driverName: string;
+    date: Date;
+    data: DriverOverviewTableData[];
+}
+
+export interface DriverOverviewTableData {
+    name: string;
+    address: {
+        line1: string;
+        line2: string;
+        town: string;
+        county: string;
+        postcode: string;
+    };
+    contact?: string;
+    packingDate: Date;
+    parcels: number | "None";
+    instructions?: string;
+}
+
+const styles = StyleSheet.create({
+    container: {
+        padding: 25,
+        alignItems: "center",
+    },
+    infoAndLogoContainer: {
+        height: "100",
+        width: "100%",
+    },
+    informationContainer: {
+        border: "2 solid black",
+        alignSelf: "flex-start",
+        margin: 10,
+        padding: 10,
+    },
+    h1text: {
+        fontSize: 24,
+        paddingLeft: 10,
+        paddingRight: 20,
+        paddingBottom: 5,
+        fontFamily: "Helvetica",
+    },
+    h2text: {
+        fontSize: 12,
+        paddingLeft: 10,
+        fontFamily: "Helvetica",
+    },
+    logoStyling: {
+        maxHeight: 60,
+        maxWidth: 60 * 1.7,
+        alignSelf: "center",
+        marginRight: 15,
+    },
+    warningSection: {
+        width: "100%",
+        paddingBottom: 10,
+    },
+    h3text: {
+        fontFamily: "Helvetica",
+        fontSize: 8,
+        paddingLeft: 15,
+        paddingTop: 5,
+    },
+    tableSection: {
+        width: "100%",
+        borderTop: "none",
+        borderBottom: "1px solid black",
+    },
+    tableRow: {
+        width: "100%",
+        fontSize: 8,
+        borderLeft: "1px solid black",
+        borderRight: "1px solid black",
+    },
+    tableColumn: {
+        padding: 5,
+        border: "1 solid black",
+        margin: 0.3,
+    },
+    flexColumn: {
+        flexDirection: "column",
+        display: "flex",
+    },
+    flexRow: {
+        flexDirection: "row",
+        display: "flex",
+        justifyContent: "space-between",
+    },
+    tableHeader: {
```

Comment:
You can probably use `flexRow` & `tableHeader` for this (unless you don't want `justifyContent: space-between`?